scripts/test_build_image.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
I'd thought it might be good to add `set -e` here but I think we don't actually set an exit status when create_images is run and profiles fail to build.